site-src/api-types/httproute.md

```diff
@@ -165,6 +165,16 @@ Service:
 Reference the [backendRef][backendRef] API documentation for additional details
 on `weight` and other fields.
 
+##### Backend Protocol
+
+??? example "Experimental Channel in v1.0.0+"
```

Comment:
this dropdown doesn't seem to work in this area - should I drop this?

Reply:
You're missing the lines below
```suggestion
??? example "Experimental Channel in v1.0.0+"

    This concept is part of the Experimental Channel in `v1.0.0+`.

```